package.json

```diff
@@ -28,11 +28,9 @@
     "e2e-local": "CYPRESS_E2E_TEST_ENV=\"local\" cypress run",
     "e2e-local-open": "CYPRESS_E2E_TEST_ENV=\"local\" cypress open",
     "format": "prettier-eslint --write 'src/**/!(*.min).{js,jsx,ts,tsx,css,json}' 'e2e_tests/**/*.{js,jsx,ts,tsx,css,json}' 'scripts/**/*.{js,jsx,ts,tsx,css,json}' 'build_scripts/**/*.{js,jsx,ts,tsx,css,json}'",
-    "jar": "yarn build && mvn package",
     "jest": "jest --coverage",
     "lint": "eslint --fix --ext .js --ext .jsx --ext .ts --ext .tsx ./",
     "precommit": "lint-staged",
-    "prepare-jar": "node ./scripts/prepare-mvn-package.js",
```

Comment:
This is used in the CI pipeline

Reply:
Oh of course, was bit too trigger-happy 😅

Reply:
Just read "jar scripts are unused" and didn't double check